### ISSUE TITLE
Acceleration versus velocity?

### ISSUE THREAD

**Issue by AlexKnauth** [oflatt/space-orbs#6]
Right now, pressing the various move keys causes your orb to move while it is pressed. To be more true to zero-g, should it instead cause acceleration while it is pressed? Afterwards the orb would be continue moving with constant velocity as it would in zero-g.

**Comment by AlexKnauth** [comment]
I can imagine a whole bunch of neat tactics using inertia and stuff.

**Comment by oflatt** [comment]
That's a great idea, and I think I will implement it in different game
modes. In the enders game gamemode, you would only be able to launch off if
walls and then have no control over movement. Another gamemode could use
that type of movement. Did you play space orbs? Some people say it does not
work on their windows machine, and they get a gl error.

**Comment by AlexKnauth** [comment]
Yes the mac version works great. P.S. I was thinking the keys would give impulses of acceleration, not continuous. That way it would feel almost exactly the same as the current version except that they would have to press one key to start moving in a direction and press the opposite key to stop.

**Comment by Nosrac42** [comment]
Works very good on Windows, but the mouse movement is very weird on mine,
but it's probably just my mouse

**Comment by oflatt** [comment]
Interesting, OK. Thanks I'll fix the spelling error.

**Comment by oflatt** [comment]
The mouse movement is weird on yours because you got a version where I was testing out a weird mouse curve. Carson, the new version has a much better mouse control.

**Comment by oflatt** [comment]
How did you come to know I released space orbs Alex?

**Comment by oflatt** [comment]
I feel that to many gamers it would feel unnatural with that kind of impulses of acceleration idea. I know it would annoy me when I played the game. It is a cool idea, and would be great to use in a future gamemode.

**Comment by AlexKnauth** [comment]
For me, I get frustrated in the constant-velocity version when I start moving at a certain speed, and then I can't go any faster. 
Also, I get frustrated in constant-acceleration games because there's a bit of a lag between holding the key and moving fast. 

But with the impulse thing, someone used to constant-velocity can do it very easily by just pressing opposite keys. I've done it before and it becomes natural very quickly. You can even decide to go twice as fast by doing the exact same thing, but with double presses.

**Comment by oflatt** [comment]
That does sound really cool, I think you have persuaded me. We should save this kind of movement so that we can go back, but I would love to try that.

**Comment by AlexKnauth** [comment]
Ok. I made an attempt to start on this here: https://github.com/AlexKnauth/space-orbs/tree/accel-sorta.

It doesn't really do what I want because the constant velocity is tied to the orb's `dir` and `current-roll` instead of the frame of reference of the room. 

But it's pretty good for being only one key-stroke away from the current behavior.
